srcpkgs/ergo/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'ergo'
+pkgname=ergo
+version=2.9.1
+revision=1
+build_style=go
+go_import_path="github.com/ergochat/ergo"
+hostmakedepends="git"
```

Comment:
git is most likely not necessary

Reply:
Will try without, I thought I saw it written to always add that for Go stuff

Reply:
Worked without